.gitignore

```diff
@@ -2,3 +2,12 @@ elm-stuff/
 node_modules/
 tmp/
 dist/
+
+#IDE configuration files
```

Comment:
agree :) It is better to enable global `.gitignore` with command `git config --global core.excludesfile ~/.gitignore_global`.